docs/building-your-bot-on-iadvize.md

```diff
@@ -343,10 +344,67 @@ Here is a full conversation example:
 | 03:43 | “Ok, i'm transferring you to a human” |                                        | ⟹           |                                                                                                                                 | POST /conversations/idConversation/messages<br>“Ok, i'm transferring you to a human"  | Do not reply to your own message                                                                                                                                            |
 |       |                                       |                                        | ⟸           | (empty replies)                                                                                                                 | Response                                                                              |                                                                                                                                                                             |
 
+### Send messages before the conversation starts
+
+This endpoint is used if you want your bot initiate the conversation with new visitors. It should return the first messages you want to send as soon as the visitor opens up the chatbox, before the first visitor message.
+
+⚠️ Please note that your bot **must** return a response in less than 2 seconds. If not, you'll receive a timeout and the bot messages will not be displayed to the user.
```

Comment:
J'aurais mis 
```suggestion
⚠️ Please note that your bot **must** return a response in less than 2 seconds. Otherwise, you'll receive a timeout and the bot messages will not be displayed to the user.
```
je trouve que ça sonne moins menaçant ^^
(si tu ne manges pas ta soupe tu n'auras pas de dessert 🤣

Reply:
Ahah bah oui je vois pas le pb :p